common/djangoapps/student/models.py

```diff
@@ -1435,3 +1435,18 @@ class DashboardConfiguration(ConfigurationModel):
     @property
     def recent_enrollment_seconds(self):
         return self.recent_enrollment_time_delta
+
+
+class LinkedInUrlConfiguration(ConfigurationModel):
+    """
+    Register Linked Add to Profile Link
+    """
+    # linked-in URL field
+    linkedin_url = models.URLField(
```

Comment:
After discussing this more with some folks, I think we're going to need to make a slight change to this configuration.  Please:

1) Rename `linkedin_url` to `dashboard_tracking_code`
2) Rename `LinkedInUrlConfiguration` to `LinkedInAddToProfileConfiguration`
3) Add a class method to this configuration that constructs the URL from the tracking code (the `?_ed` parameter).  You can call this class method in `student.views` to get the full URL.

The reason why we need to make this change is that eventually we may need to specify different tracking codes in different places (e.g. one tracking code for the dashboard, another for emails we send out).

Sorry for the last minute change -- I didn't fully appreciate that we might need different tracking codes in different places until this afternoon.